pylutron/__init__.py

```diff
@@ -853,13 +855,91 @@ def __init__(self, lutron, name, integration_id):
     self._integration_id = integration_id
 
 
+class OccupancyGroup(LutronEntity):
+  _CMD_TYPE = 'GROUP'
+  _ACTION_STATE = 3
+
+  OCCUPIED = 3
+  VACANT = 4
+  UNKNOWN = 255
+
+  class Event(LutronEvent):
+    """OccupancyGroup event that can be generated.
+
+    OCCUPANCY_CHANGED: Occupancy state has changed.
+        Params:
+          state: One of OCCUPIED/VACANT/UNKNOWN
+    """
+    OCCUPANCY = 1
+
+  def __init__(self, lutron, area):
+    super(OccupancyGroup, self).__init__(lutron, 'Occ {}'.format(area.name))
+    self._area = area
+    self._integration_id = area.id
+    self._state = OccupancyGroup.UNKNOWN
+    self._lutron.register_id(OccupancyGroup._CMD_TYPE, self)
+    self._query_waiters = _RequestHelper()
+
+  @property
+  def id(self):
+    """The integration id"""
+    return self._integration_id
+
+  @property
+  def name(self):
+    """Return the name of this OccupancyGroup, which is 'Occ' plus the name of the area."""
+    return 'Occ {}'.format(self._area.name)
+
+  @property
+  def state(self):
+    """Returns the current occupancy state."""
+    # Poll for the first request.
```

Comment:
It will always poll here. Did you also mean to add a separate interface to maintain a cached version?

Reply:
Does it? I would think after the first call the if statement won't be true because it's getting updated via the callback. The poll only happens the first call when no callbacks have been issued yet.